src/store/getters.ts

```diff
@@ -76,6 +77,16 @@ export const getJurisdiction = (state: StateIF): string => {
   return state.stateModel.newRequestModel.request_jurisdiction_cd
 }
 
+export const getJurisdictionText = (state: StateIF): string => {
```

Comment:
This is not really a getter. Should be in an external mixin (assuming said mixin can use a `@Getter` to get the location).